### DISCUSSION TITLE
Ability to archive a shipping or tax category

### DISCUSSION THREAD

**Discussion by lukeholder** [craftcms/commerce#2431] [Ideas]
Add a `dateDeleted` to shipping and tax categories so they can still be related to old order line items.

**Comment by rob-c-baker** [comment]
This: https://github.com/craftcms/commerce/issues/1551 links to https://github.com/craftcms/commerce/issues/1815 which points here. Any news on when we might see something on this?

**Reply by lukeholder** [reply]
We have a PR for it that needs updating, but we will regroup on it and get it out for the next 3.x release:

https://github.com/craftcms/commerce/pull/1915
